pkg/lib/authenticationflow/declarative/node_do_create_user.go

```diff
@@ -20,18 +20,23 @@ var _ authflow.Milestone = &NodeDoCreateUser{}
 var _ MilestoneDoUseUser = &NodeDoCreateUser{}
 var _ MilestoneDoCreateUser = &NodeDoCreateUser{}
 var _ authflow.EffectGetter = &NodeDoCreateUser{}
-var _ MilestoneSwitchToUser = &NodeDoCreateUser{}
+var _ MilestoneSwitchToExistingUser = &NodeDoCreateUser{}
 
 func (n *NodeDoCreateUser) Kind() string {
 	return "NodeDoCreateUser"
 }
 
-func (*NodeDoCreateUser) Milestone()                      {}
-func (n *NodeDoCreateUser) MilestoneDoUseUser() string    { return n.UserID }
-func (n *NodeDoCreateUser) MilestoneDoCreateUser() string { return n.UserID }
-func (n *NodeDoCreateUser) MilestoneSwitchToUser(newUserID string) {
+func (*NodeDoCreateUser) Milestone()                   {}
+func (n *NodeDoCreateUser) MilestoneDoUseUser() string { return n.UserID }
+func (n *NodeDoCreateUser) MilestoneDoCreateUser() string {
```

Comment:
Now that this method could return an empty string, it is better to change the signature to `(string, bool)` so that the caller is aware of the possibility of the return of an empty string.